db/migrations/20230104051118_create_potsColors_table.sql

```diff
@@ -0,0 +1,10 @@
+-- migrate:up
+CREATE TABLE potsColors (
```

Comment:
```suggestion
CREATE TABLE colors (
```
중간테이블이 아니기에 단순 colors로 지어주는 것이 더 명확할 것 같습니다.